rero_ils/modules/circ_policies/jsonschemas/circ_policies/circ_policy-v0.0.1.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "CircPolicy",
+  "description": "JSON schema for circ_policies.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "name",
+    "description",
+    "organisation_pid"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate circ_policies records against.",
+      "type": "string",
+      "minLength": 9
+    },
+    "pid": {
+      "title": "CircPolicy ID",
+      "type": "string",
+      "minLength": 1
+    },
+    "name": {
+      "title": "CircPolicy name",
+      "description": "The name of the circulation policy",
+      "type": "string",
+      "minLength": 2
+    },
+    "description": {
+      "title": "CircPolicy description",
+      "description": "The description of the circulation policy",
+      "type": "string"
+    },
+    "organisation_pid": {
+      "title": "Organisation ID",
+      "description": "The organisation_pid pid of this circulation policy",
```

Comment:
`The PID of the organisation in which this circulation policy applies.` Or something like it.